test/e2e/framework/util.go

```diff
@@ -294,6 +294,19 @@ func Skipf(format string, args ...interface{}) {
 	ginkgowrapper.Skip(nowStamp() + ": " + msg)
 }
 
+// SkipUnlessLocalSSDExists takes in an ssdInterface (scsi/nvme) and a filesystemType (fs/block)
+// and skips if a disk of that type does not exist on the node
+func SkipUnlessLocalSSDExists(ssdInterface, filesystemType string, node *v1.Node) {
```

Comment:
I would keep this function in the local volume e2e test because it's unlikely other e2e tests will need it.